java/ql/src/Likely Bugs/Comparison/NoComparisonOnFloats.ql

```diff
@@ -61,7 +51,7 @@ predicate similarVarComparison(EqualityTest e) {
 
 from EqualityTest ee
 where
-  ee.getAnOperand().getType() instanceof Floating and
+  ee.getAnOperand().getType() instanceof FloatingPointType and
```

Comment:
This change it not actually directly related, but I noticed it while searching for usages of the old expression name.
If you want I can revert this change.